api_spec.yaml

```diff
@@ -322,12 +323,18 @@ paths:
                       $ref: '#/components/schemas/createJWTPayload'
                   required:
                     - authorizationCode
+                    - clientId
                 - type: object
                   properties:
                     clientId:
                       type: string
+                      $ref: '#/components/schemas/oauth2ClientId'
```

Comment:
- `UNKNOWN_AUTH_CODE_ERROR ` notifies that the auth_code is not found.
-`INVALID_AUTHORIZATION_CODE_ERROR ` is to notify that the auth code is expired. also alternative to this would  be,`EXPIRED_AUTHORIZATION_CODE_ERROR`

Reply:
As discussed , 
- UNKNOWN_AUTH_CODE_ERROR  means that the auth_code is not found.
- AUTH_CODE_EXPIRY_ERROR means that auth code is expired.